Automattic/wp-calypso
# Composite-checkout: Remove "...UI" pattern for styled components

#### Changes proposed in this Pull Request

Previously we had been using the "...UI" suffix on React components which were styled using the CSS-in-JS emotion package, particularly when those components' names would clash with the unstyled versions. However, this was never documented and is unclear and inconsistently applied. Moreover, those components were never meant to be part of a public API.

In this PR, we rename all such components to remove that suffix. Mostly this is inside the `@automattic/composite-checkout` package, but also in calypso and in the `editing-toolkit` app (see https://github.com/Automattic/wp-calypso/pull/44821).

Fixes https://github.com/Automattic/wp-calypso/issues/45680

#### Testing instructions

There are three things that will need testing: the composite-checkout package, checkout in calypso, and whatever the editing-toolkit uses these components for.

1. The `@automattic/composite-checkout` package.
- Run `yarn run composite-checkout-demo`.
- Open the demo page and complete the checkout form.
- Verify that there are no errors and that everything looks decent.

2. Calypso checkout.
- Start calypso or use calypso.live.
- Visit checkout with a product in the cart.
- Complete the checkout process.
- Verify that there are no errors and that everything looks decent.

3. Site setup flow in `editing-toolkit` plugin
- Apply D50672-code to your sandbox.
- Create a site starting from [wordpress.com/new](https://www.wordpress.com/new).
- Sandbox the site.
- Click _Complete Setup_ in editor,
- When reaching the the last step in the sidebar (_Launch your site_), the design should match screenshots in https://github.com/Automattic/wp-calypso/pull/44821 description.
